### PR TITLE
Add Image Inpainting; Reform readme doc.

### PR DESCRIPTION
**Pull request by SamitHuang**
1. Add text-guided image inpainting inference pipeline, supporting two models:
 - sd-2-inpainting
 - sd-1.5 cn (wukong) inpainting

For detailed instructions and results, refer to README.md

2. Reform README for better organization and text description. ** Please view README.md file and welcome advices**

### CODE REVIEW

File: examples/stable_diffusion_v2/README.md [hunk 1]
```diff
@@ -94,107 +101,212 @@ While `--prompt` indicates what to render in the generated images, the negative
...
+
+To train SD 1.5 on a custom text-image dataset, please run
 
 ```shell
+# SD 1.5 vanilla training
 sh scripts/run_train_v1.sh
 ```
+after setting `data_path` in `run_train_v1.sh` to your dataset path.
+
+> Note: to run other training pipelines on SD 1.5, you can refer to training tutorials of SD 2.0 and change the following arguments in the trainin script: set `--model_config` argument to `configs/v1-train.yaml`, `--train_config` to `configs/train_config.json`, and set `--ckpt_path` to `models/ms_v1_5_pruned_emaonly-d0ab7146.ckpt`.
```
Comment (DongyaoZhu) [inline examples/stable_diffusion_v2/README.md:248]: trainin-training

File: examples/stable_diffusion_v2/README.md [hunk 2]
```diff
@@ -94,107 +101,212 @@ While `--prompt` indicates what to render in the generated images, the negative
   <em> With negative prompt: "moss" </em>
 </p>
 
-#### 1.2 Supported Diffusion Process Schedulers
+### Text-guided Image Inpainting
```
Comment (DongyaoZhu) [inline examples/stable_diffusion_v2/README.md:104]: maybe add a short description of how it works (img - mask - forward - reverse)?

**Review by DongyaoZhu — APPROVED** [review]
weak accept

**Comment by SamitHuang** [comment]
Thanks for the advice. PR updated accordingly.

**Pull request merged**